Formula/hwatch.rb

```diff
@@ -0,0 +1,22 @@
+class Hwatch < Formula
+  desc "Modern alternative to the watch command"
+  homepage "https://github.com/blacknon/hwatch"
+  url "https://github.com/blacknon/hwatch/archive/refs/tags/0.3.5.tar.gz"
+  sha256 "e01bf716db8d1561f79d12c178df3b48e927957c6c595b841ba2b1bc4faa760e"
+  license "MIT"
+  head "https://github.com/blacknon/hwatch.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+    man1.install "man/hwatch.1"
+    bash_completion.install "completion/bash/hwatch-completion.bash"
+    zsh_completion.install "completion/zsh/_hwatch"
+    fish_completion.install "completion/fish/hwatch.fish"
+  end
+
+  test do
+    assert_match "hwatch #{version}", shell_output("#{bin}/hwatch --version")
```

Comment:
Since this is a modern alternative to `watch`, it would be re-use the test from `watch` formula.

Reply:
Unfortunately, we can't.

`watch` has options such as `--errexit` (exit on command error) or `--chgexit` (exit on change), but `hwatch` doesn't.

Reply:
Maybe I should suggest implementing those features to the author.

Reply:
For now, I just use a spawned thread to run the hwatch, it seems working fine.

Reply:
yeah, can you create some feature request for `hwatch` author to make it on par with `watch`? Thanks!